sw/device/tests/crypto/cryptotest/firmware/ibex_fi.S

```diff
@@ -0,0 +1,228 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Increments a value 100 times by 1.
+ *
+ * @param a0 initial value which gets incremented.
+ */
+  .globl target_function
+  .type target_function, @function
+  .balign 256
+target_function:
```

Comment:
perhaps these could have better names, e.g., `increment_100x1` and `increment_100x10`?